src/components/ChatLog.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+  const chatEntryComponents = props.entries.map((entry) => {
+    return (
+      <ChatEntry
+        key={entry.id}
+        id={entry.id}
+        sender={entry.sender}
+        body={entry.body}
+        timeStamp={entry.timeStamp}
+        liked={entry.liked}
+        toggleHeartCallback={props.toggleHeartCallback}
```

Comment:
Since you use this `toggleHeartCallback` prop, it's likely a good idea to include it in your `PropTypes` below.

I would even recommend that it be `.isRequired` because this callback is pretty vital to the function of the code. However, that would cause issues with our tests, that do not supply this callback, so it'd be OK without `.isRequired`.